experimental/PropertyDDS/examples/partial-checkout/package.json

```diff
@@ -43,8 +43,8 @@
     "@fluidframework/aqueduct": "^0.42.0-0",
     "@fluidframework/common-utils": "^0.31.0",
     "@fluidframework/container-definitions": "^0.40.0-0",
-    "@fluidframework/container-loader": "^0.42.0",
-    "@fluidframework/container-runtime": "^0.42.0",
+    "@fluidframework/container-loader": "^0.42.0-0",
```

Comment:
I thought this is not needed, as examples are part of same mono-build as runtime. Did it change?

Reply:
I'm pretty sure it's not needed, as we do not publish pre-release builds for runtime (so that should fail install).
Please correct me if I got it wrong.

Reply:
Hmm.. I just did a blind replace all for the `"@fluidframework/container-definitions"` dependency in vscode :-s. I thought lerna was going to complain about different dependency versions.